spec/spec_helper.rb

```diff
@@ -1,2 +1,4 @@
+require "coveralls"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.